### PR TITLE
chore: Social sign in with web ui page for flutter

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Adds social sign in with web ui page for Flutter.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Ashish-Nanda — APPROVED** [review]
LGTM 🚀

**Review by fjnoyp — APPROVED** [review]

File: docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+In terminal, navigate to your project, run `amplify add auth`, and choose the following options (the last steps are specific to Facebook here but are similar for other providers):
```
Comment (richardmcclellan) [inline docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md:1]: Which steps are specific to Facebook?  I don't see any below.

File: docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+? Do you want to use the default authentication and security configuration? 
+    `Default configuration with Social Provider (Federation)`
+? How do you want users to be able to sign in? 
+    `Username`
+? Do you want to configure advanced settings? 
+    `No, I am done.`
+? What domain name prefix you want us to create for you? 
+    `(default)`
+? Enter your redirect signin URI: 
+    `myapp://`
```
Comment (richardmcclellan) [inline docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md:13]: The Android docs say this should be `myapp://callback/`, and the redirect signout URI below should be ``myapp://signout/`.  Although, the iOS docs seem to make what you have here for Flutter.   Are you sure this is correct?  Does it matter?

File: docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md (hunk 1 above)
Comment (haverchuck) [reply docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md:1]: This was taken directly from the native docs. In any case, I 'm going to remove here.

File: docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md (hunk 2 above)
Comment (haverchuck) [reply docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md:13]: In iOS, the user is navigated back to wherever they launched the OAuth flow. We basically mimic them - on the Android side by navigating back to the  Android Activity that is created as part of our Flutter plugin.

**Review by richardmcclellan — APPROVED** [review]

**Pull request merged**